### DISCUSSION TITLE
Is there any way to use shortcuts when the page is in the background

### DISCUSSION THREAD

**Discussion by eaglexiang** [navidrome/navidrome#2296] [Q&A]
It's not suitable to set Navidrome in frontground all the time, so can I use the user friendly shortcuts with navidrome in background

**Comment by deluan** [comment]
I may be wrong, but I think this is a limitation of the browser, and you'd need external tools to accomplish that. Navidrome already shows the [MediaSession](https://github.com/lijinke666/react-music-player#bulb-media-session), maybe there's a plugin for your browser that attaches hotkeys to it?
